tests/modules/subcommands.c

```diff
@@ -35,12 +35,35 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     if (RedisModule_Init(ctx, "subcommands", 1, REDISMODULE_APIVER_1) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    int count = 0;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char\r",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char\n",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char ",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char|",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char@",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char=",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char:",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char,",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (count != 8) return REDISMODULE_ERR;
```

Comment:
it may be excessive to test all of them, and even do that twice.
but either way, please add a short comment explaining this block.